src/config/areas.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'versions' => function () {
+        return [
+            'label' => t('view.versions'),
+            'icon'  => 'layers',
+            'menu'  => true,
```

Comment:
And isn't the `disabled` value still supported in 3.6, @bastianallgeier? If so, the old Js statement could be migrated over to here.

Reply:
Yep, that works 1:1. 👍